spec/create_release_backmerge_pull_request_spec.rb

```diff
@@ -301,26 +309,32 @@ def stub_expected_pull_requests(expected_backmerge_branches:, source_branch:, la
   end
 
   context 'when the branch already exists' do
-    it 'does not try to create a new branch' do
-      stub_git_release_branches(%w[release/30.6])
+    it 'delete the branch before creating it anew' do
+      stub_git_release_branches(%w[release/30.7])
 
       source_branch = 'release/30.7'
 
-      expected_backmerge_branches = %w[trunk release/30.6]
-      stub_expected_pull_requests(
-        expected_backmerge_branches: expected_backmerge_branches,
+      intermediate_branches = stub_expected_pull_requests(
+        expected_backmerge_branches: [default_branch],
         source_branch: source_branch,
         branch_exists_on_remote: true
       )
 
-      expect do
-        run_described_fastlane_action(
-          github_token: test_token,
-          repository: test_repo,
-          source_branch: source_branch,
-          target_branches: expected_backmerge_branches
-        )
-      end.to raise_error(FastlaneCore::Interface::FastlaneError, 'The intermediate branch `merge/release-30.7-into-trunk` already exists. Please check if there is an existing Pull Request that needs to be merged or closed first, or delete the branch.')
+      intermediate_branches.each do |branch|
+        expect(FastlaneCore::UI).to receive(:important)
+          .with("An intermediate branch `#{branch}` already existed on the remote. It will be deleted and any associated existing PR will be closed.")
```

Comment:
If we go with my suggestion above,

```suggestion
          .with("An intermediate branch `#{branch}` already exists on the remote. It will be deleted and GitHub will close any associated existing PR.")
```